examples/developer-tools/src/storage.ts

```diff
@@ -113,3 +115,22 @@ export const getAllSavedBlockNames = function (): Set<string> {
 export const getProhibitedBlockNames = function (): Set<string> {
   return prohibitedBlockNames;
 };
+
+export interface BlockFactorySettings {
+  blockDefinitionFormat: string;
```

Comment:
You could make these enums to catch some buggies. But this code is simple enough I don't think it's really necessary.

Reply:
I started to, but since I get the values from the dom, it added a lot of boilerplate and some casting to convert the raw string out of the dom into the enum. It didn't feel worth the overhead but someone can link to this comment in the future when they discover a bug here :P